frontend/src/app/auth/callback/page.tsx

```diff
@@ -17,53 +17,57 @@ export default function AuthCallbackPage() {
       const supabase = createClient();
       const { data: { session }, error } = await supabase.auth.getSession();
 
-      // Get the hash and search params from the URL
-      const hash = window.location.hash;
+
+      // If we have a session, redirect to home
+      if (session) {
+        setMessage("Account is verified! Redirecting to homepage...");
```

Comment:
should this be `"Authentication successful! Redirecting to homepage..."` ?

Reply:
updated